.github/workflows/scheduled-tasks.yml

```
@@ -0,0 +1,284 @@
+          done
+          
+          if [ $FAILED -gt 0 ]; then
+            echo "failed=$FAILED" >> $GITHUB_OUTPUT
+            exit 1
+          fi
+
+      - name: Check critical services
+        run: |
+          # Check database connectivity
+          ./scripts/check-database.sh
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)